app/queries/get_content_collection.rb

```diff
@@ -36,6 +40,10 @@ def content_items
 
       live_items = live_items.where(publishing_app: @publishing_app) if @publishing_app.present?
 
+      @draft_versions = Version.in_bulk(draft_items, DraftContentItem)
```

Comment:
@danielroseman I am not 100% sure, but would it be possible to add `belongs_to :version` on `LiveContentItem`/`DraftContentItem`, so that we can use `includes(:version)` and `item.version` instead of `Version.in_bulk`?